index.js

```diff
@@ -7,13 +7,34 @@ const through = require('through2')
 
 module.exports = JSONToCsv
 
-function JSONToCsv () {
-  return pumpify(JSONStream.parse(), split(), CsvStream())
+const csvOpts = ['separator', 'newline', 'headers', 'sendHeaders']
+
+function JSONToCsv (options) {
+  options = options || {}
+  return pumpify(
+    JSONStream.parse(options.path),
+    split(),
+    CsvStream(pick(csvOpts, options))
+  )
 }
 
 function split () {
-  return through.obj(function (array, enc, callback) {
-    array.forEach(this.push.bind(this))
+  return through.obj(function (chunk, enc, callback) {
+    if (isArray(chunk)) chunk.forEach(this.push.bind(this))
```

Comment:
I was aiming to node versions prior to 6.4. `Array.isArray` doesn't work with previous versions. http://node.green/#ES2015-built-ins-Proxy-Array-isArray-support